.eslintrc.js

```diff
@@ -72,6 +75,6 @@ module.exports = {
     'import/no-dynamic-require': 0,
     quotes: [2, 'single', { avoidEscape: true }],
     'arrow-body-style': 0,
-    allowElseIf: true,
```

Comment:
Just confirming this diff is desired?

Reply:
Yes, it seemed like eslint didn't recognize the `allowElseIf` rule and I think the equivalent is setting `no-else-return: 0`